src/cljs/nr/gameboard.cljs

```diff
@@ -1470,6 +1479,9 @@
             [:div
              [:a {:on-click #(close-popup % (:hand-popup @s) nil false false)} (tr [:game.close "Close"])]
              [:label (tr [:game.card-count] size)]
+             (let [{:keys [total]} @hand-size]
+               [:div.hand-size (str total " " (tr [:game.max-hand "Max hand size"]))
+                (controls :hand-size)])
```

Comment:
Does this need a `(when me?` check?

Reply:
It's inside a `(when popup`, which is only `true` for your side as far as I can see. There's the special case with spectators, but they see (non-functional) controls now already.

Reply:
Cool, thanks!